macros/utils/graph/get_model_database_and_schema.sql

```diff
@@ -0,0 +1,15 @@
+{%- macro get_model_database_and_schema(package_name, model_name) -%}
```

Comment:
Does this work for models that are not from a package as well?
It is unclear. 
If so - can you add a comment and change the order of params?
If not - the name is not indicative.